bundles/azure-bundle/build.gradle.kts

```diff
@@ -25,7 +25,10 @@ plugins {
 }
 
 dependencies {
-  implementation(project(":bundles:azure"))
+  implementation(project(":bundles:azure")) {
+    exclude(group = "commons-logging", module = "commons-logging")
```

Comment:
Why exclude this? You left a comment saying it is needed for runtime.

https://github.com/apache/gravitino/blob/728ea2e702177cd0dc456ba5e525795d14e67f5d/bundles/gcp/build.gradle.kts#L44-L45

Reply:
If so, could you plz add this to the code comment?